scripts/run_tests.sh

```diff
@@ -32,21 +32,25 @@ function display_result {
   fi
 }
 
-flake8 --max-complexity 10 --count
-display_result $? 1 "Code style check"
 
-py.test --cov=app --cov-report xml $@ $1
-display_result $? 2 "Unit tests"
+if [ -z "$EQ_FUNCTIONAL_TEST_SUITES" ]; then
+    export EQ_FUNCTIONAL_TEST_SUITES="core"
+fi
 
-# Run front end tests
-npm config set python python2.7
+if [ "$EQ_FUNCTIONAL_TEST_SUITES" = "core" ]; then
```

Comment:
If a unit test fails then the other builds will run all of the functional tests? surely this breaks the fail fast rule?